File: Question 01/removing.js
// To removejunk or special character in a string.
function answer(s) {
    let result = '';
    
 
    for (let i = 0; i < s.length; i++) {
        let charCode = s.charCodeAt(i);
        
        if ((charCode >= 48 && charCode <= 57) || (charCode >= 65 && charCode <= 90) || (charCode >= 97 && charCode <= 122)) {
            result += s[i];  
        }
    }
    
    return result;
}

File: Question 01/removing.py
def answer(s):
    result = ""
    
    for char in s:
        char_code = ord(char)
    
        if (48 <= char_code <= 57) or (65 <= char_code <= 90) or (97 <= char_code <= 122):
            result += char
    
    return result